documentation/src/docs/pravega-write-read-methods.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Methods for Writing and Reading Pravega Streams
+---
+
+import { IfHaveFeature, IfMissingFeature, SDPVersion } from 'nautilus-docs';
+
+## Overview
+
+This table describes the methods that can be used to write and read data in Pravega streams. It is intended to be a comprehensive list which includes open-source, experimental, and commercial products that integrate with Pravega.
+
+Method                                                                                          | Write | Transactional Write | Read | Special Features | Notes
+------------------------------------------------------------------------------------------------|-------|---------------------|------|------------------|------------------------------------------------------
+[Pravega Sensor Collector](#pravega-sensor-collector)                                           | Yes   | Yes                 | No   | EO               |
+[SDP Ingest Gateway](#sdp-ingest-gateway)                                                       | Yes   | No                  | No   |                  |
+[Pravega Ingest Gateway](https://github.com/pravega/pravega-ingest-gateway)                     | Yes   | No                  | Yes  |                  |
+[Pravega Search](#pravega-search)                                                               | Yes   | Yes                 | Yes  | EO               |
```

Comment:
Is this private or publicly available?

Reply:
Like the SDP Ingest gateway, this is a supposed to be disabled.

This page comes from new SDP docs, and I brought it in because it is generally useful once the proprietary bits are squashed.

We want to have a sensible way of including/suppressing/extending docs for the sake of systems that build on top of, re-use or extend Pravega, so SDP docs are a bit of a test case in this regard.

Reply:
I've restored the PSearch description that was lost in an intermediate PR. @claudiofahey intended to list open, experimental, and commercial resources here.